console.log("hello");
console.log("hello");
// let a=3;
try {
  console.log(a);
} catch (err) {
  console.log("Caught an error... Variable a doesn't exist.");
  
//   !To print error ðŸ‘‡
  console.log(err);
}

console.log("hello2");
console.log("hello2");
console.log("hello2");
